HotKey/HotKey_MainFolder/HotKeyControl.cs

```diff
@@ -52,16 +52,22 @@ private void KeybindButton_KeyUp(object sender, KeyEventArgs e)
                 if (e.Shift)
                     modKeysValue += 4;
 
-                if (hotKeyItem.SetKeybind((ModKeys)modKeysValue, key))
-                    SetKeybindText(hotKeyItem);
-                else
-                    keybindButton.Text = "Not Bound";
+                UpdateKeybind(key, (ModKeys) modKeysValue);
             }
+            //undo/reset keybind if escape key pressed
+            else if (key == Keys.Escape)
```

Comment:
I like this a lot better than pressing the same keybind again. Good choice. Maybe we should just have it where if someone clicks on it again it will just unset it anyways?